datalake/src/main/java/com/sequenceiq/datalake/flow/upgrade/database/event/SdxUpgradeDatabaseServerSuccessEvent.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.datalake.flow.upgrade.database.event;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.datalake.flow.SdxEvent;
+
+public class SdxUpgradeDatabaseServerSuccessEvent extends SdxEvent {
+
+    @JsonCreator
+    public SdxUpgradeDatabaseServerSuccessEvent(
+            @JsonProperty("resourceId") Long sdxId,
+            @JsonProperty("userId") String userId) {
+        super(sdxId, userId);
+    }
+
+    @Override
+    public String selector() {
+        return SdxUpgradeDatabaseServerSuccessEvent.class.getSimpleName();
```

Comment:
same here, please don't override this

Reply:
Yes, I corrected all these. I actually copied another compact flow from SDX flows instead of building a new one from scratch. All you write makes sense. Thank you.